manager-bundle/src/HttpKernel/ContaoKernel.php

```diff
@@ -319,17 +320,21 @@ private function getConfigFile(string $file): ?string
         $rootDir = $this->getProjectDir();
 
         foreach (['.yaml', '.yml'] as $ext) {
-            if (file_exists($rootDir.'/config/'.$file.$ext)) {
-                return $rootDir.'/config/'.$file.$ext;
+            $path = Path::join($rootDir, 'config', $file.$ext);
+
+            if (file_exists($path)) {
+                return $path;
             }
         }
 
         // Fallback to the legacy config file (see #566)
         foreach (['.yaml', '.yml'] as $ext) {
-            if (file_exists($rootDir.'/app/config/'.$file.$ext)) {
+            $path = Path::join($rootDir, 'app/config', $file.$ext);
+
+            if (file_exists($path)) {
```

Comment:
```suggestion
            if (file_exists($path = Path::join($rootDir, 'app/config', $file.$ext))) {
```

Reply:
see https://github.com/contao/contao/pull/1902#discussion_r450123027

Reply:
We had to revert the changes for now due to https://github.com/slevomat/coding-standard/pull/1053.